app/bundles/PageBundle/Translations/en_US/messages.ini

```
@@ -33,6 +33,8 @@ mautic.page.config.form.google.analytics="Analytics script (i.e. Google Analytic
 mautic.page.config.form.google.analytics.tooltip="Insert the analytics script to have it automatically included in the soure of landing pages."
```

This was updated in 5f9139c, and fails against [release:2.15.3](/mautic/mautic/tree/2.15.3).

```suggestion
mautic.page.config.form.google.analytics.tooltip="Insert the analytics script to have it automatically included in the source of landing pages."
```

Has been working well on 2.15.1/2.15.2

---

I think the PR still needs a rebase.

[Mautibox agrees](https://mautibox.com/7614):
> Patch could not be applied. Try rebasing the branch for this pull request.